frame/randomness-collective-flip/README.md

```diff
@@ -20,18 +20,28 @@ the system trait.
 ### Example - Get random seed for the current block
 
 ```rust
-use frame_support::{decl_module, dispatch, traits::Randomness};
-
-pub trait Config: frame_system::Config {}
-
-decl_module! {
-	pub struct Module<T: Config> for enum Call where origin: T::Origin {
-		#[weight = 0]
-		pub fn random_module_example(origin) -> dispatch::DispatchResult {
-			let _random_value = <pallet_randomness_collective_flip::Module<T>>::random(&b"my context"[..]);
-			Ok(())
-		}
-	}
+use frame_support::traits::Randomness;
+
+#[frame_support::pallet]
+pub mod pallet {
+    use super::*;
+    use frame_support::pallet_prelude::*;
+    use frame_system::pallet_prelude::*;
+
+    #[pallet::pallet]
+    pub struct Pallet<T>(_);
+
+    #[pallet::config]
+    pub trait Config: frame_system::Config + pallet_randomness_collective_flip::Config {}
```

Comment:
The declarative macros don't seem to indicate that this Config trait needs to have `pallet_randomness_collective_flip::Config` as a supertrait.

Reply:
the function `random` exists for struct `pallet_randomness_collective_flip::Pallet<T>`, which needs `T: pallet_randomness_collective_flip::Config` to be the trait bound